backend/app/api/routes/openai_conversation.py

```
@@ -0,0 +1,170 @@
+from typing import List
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)